Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+  return (
+    <a href="https://reactjs.org/">
+      <img className="react-img" src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" alt="react image"></img>
```

Comment:
having the work *image* in the alt attribute prompts the following warning
*Line 7:  Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don’t need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop  jsx-a11y/img-redundant-alt*
Simply changing *image* to *logo* should avoid this warning.